CMakeLists.txt

```diff
@@ -32,7 +32,7 @@ find_package(ZLIB REQUIRED) #framework
 find_package(UUID REQUIRED) #framework
 find_package(CURL REQUIRED) #framework, etcdlib
 find_package(LIBZIP REQUIRED) #framework, etcdlib
-find_package(Jansson REQUIRED) #etcdlib, dfi
+find_package(jansson REQUIRED) #etcdlib, dfi
```

Comment:
nitpick: message("Note jansson::jansson target not created by find_package(Jansson). Creating jansson::jansson Target.") should also be updated.